service/src/main/java/datawave/microservice/authorization/AuthorizationControllerV2.java

```diff
@@ -0,0 +1,172 @@
+package datawave.microservice.authorization;
+
+import datawave.microservice.authorization.user.DatawaveUserDetails;
+import datawave.security.authorization.CachedDatawaveUserService;
+import datawave.security.authorization.DatawaveUser;
+import datawave.security.authorization.DatawaveUserInfo;
+import datawave.user.AuthorizationsListBase;
+import io.swagger.v3.oas.annotations.ExternalDocumentation;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+
+import static datawave.microservice.http.converter.protostuff.ProtostuffHttpMessageConverter.PROTOSTUFF_VALUE;
+
+/**
+ * Presents the REST operations for the authorization service. This version returns the updated (V2) DatawaveUser
+ */
+@Tag(name = "Authorization Operations /v2",
```

Comment:
I don't see any difference in implementations between V2 and V1.

Reply:
The rest interface is the same, but the content of the responses (i.e. ordering of the proxied entities) is different.  FWIW, this  already exists on main - I just separated the rest controllers from the logic.